VideoLocker/res/values/strings.xml

```diff
@@ -73,7 +73,7 @@
     <!-- Alert dialog okay button -->
     <string name="play_data_dialog_OK">OK</string>
     <!-- Alert dialog cancel button -->
-    <string name="play_data_dialog_Cancel">Cancel</string>
+    <string name="play_data_dialog_Cancel">@android:string/cancel</string>
```

Comment:
I think these strings can simply be removed from here, and the framework strings referenced directly where these are used.